migrations/sql/V2021.06.16.16.27__populate_camp_detail.sql

```diff
@@ -0,0 +1,5 @@
+-- The new camp_detail table has an FK to activity_detail_ table. Ppulate the new table with exisiting activity_detail data
```

Comment:
Typo in comment "Ppulate" and "activity_detail_"

Reply:
fixed